src/test/org/firebirdsql/gds/ng/wire/version10/TestV10Database.java

```diff
@@ -434,7 +434,8 @@ private FbTransaction getTransaction(FbDatabase db) throws SQLException {
     private static void safelyClose(FbDatabase db) {
         if (db == null) return;
         try {
-            db.close();
+            if (db.isAttached())
```

Comment:
Why this change?

Reply:
To avoid 'is not attached to a database' message.